src/views/network/view_network.cpp

```diff
@@ -441,53 +473,21 @@ namespace big
 		    true,
 		    "Teleport");
 
-		ImGui::EndGroup();
-
-
-		components::command_button<"sextall">({}, "Send Sexts");
-		ImGui::SameLine();
-		components::command_button<"fakebanall">({}, "Send Fake Ban Messages");
-
-		components::small_text("WARP_TIME"_T.data());
-
-		components::button("PLUS_1_MINUTE"_T, [] {
-			toxic::warp_time_forward_all(60 * 1000);
-		});
-		ImGui::SameLine();
-		components::button("PLUS_5_MINUTES"_T, [] {
-			toxic::warp_time_forward_all(5 * 60 * 1000);
-		});
-		ImGui::SameLine();
-		components::button("PLUS_48_MINUTES"_T, [] {
-			toxic::warp_time_forward_all(48 * 60 * 1000);
-		});
-		ImGui::SameLine();
-		components::button("PLUS_96_MINUTES"_T, [] {
-			toxic::warp_time_forward_all(96 * 60 * 1000);
-		});
-		ImGui::SameLine();
-		components::button("PLUS_200_MINUTES"_T, [] {
-			toxic::warp_time_forward_all(200 * 60 * 1000);
-		});
-		ImGui::SameLine();
-		components::button("STOP_TIME"_T, [] {
-			toxic::set_time_all(INT_MAX - 3000);
-		});
-		if (ImGui::IsItemHovered())
-			ImGui::SetTooltip("STOP_TIME_DESC"_T.data());
-
 		components::sub_title("SCRIPT_HOST_FEATURES"_T);
 		ImGui::Checkbox("DISABLE_CEO_MONEY"_T.data(), &g.session.block_ceo_money);
 		if (ImGui::IsItemHovered())
 			ImGui::SetTooltip("DISABLE_CEO_MONEY_DESC"_T.data());
 		ImGui::SameLine();
-		ImGui::Checkbox("RANDOMIZE_CEO_COLORS"_T.data(), &g.session.randomize_ceo_colors);
 		ImGui::Checkbox("Block Jobs", &g.session.block_jobs);
+
+		ImGui::Checkbox("Random CEO Colors", &g.session.randomize_ceo_colors);
 		if (ImGui::IsItemHovered())
 			ImGui::SetTooltip("Prevents remote players from starting jobs while in your session");
```

Comment:
What does Random CEO colors have to do with the description set by IsItemHovered?